scipy/constants/_constants.py

```diff
@@ -225,7 +225,7 @@
 # functions for conversions that are not linear
 
 
-@xp_capabilities()
+@xp_capabilities(static_argnames=("old_scale", "new_scale"))
```

Comment:
I had an offline conversation with @pearu regarding the opportunity to automatically wrap these functions with `jax.jit` _in production_, and not just in unit tests, whenever we detect an jnp.Array argument.

The outcome was that, while you can observe very substantial benefits compared to running eagerly, the jit has a cost every time the shape of the input array changes, so it may be detrimental whenever someone iterates on ever-differently-shaped arrays.